ray-operator/config/samples/tpu/tpu_list_devices.py

```diff
@@ -0,0 +1,19 @@
+import ray
+
+ray.init(
+    runtime_env={
+        "pip": [
+            "jax[tpu]==0.4.33",
+            "-f https://storage.googleapis.com/jax-releases/libtpu_releases.html",
+        ]
+    }
+)
+
+@ray.remote(resources={"TPU": 4})
+def tpu_cores():
+    import jax
+    return "TPU cores:" + str(jax.device_count())
+
+num_workers = int(os.environ['TPU_HOSTS']) # Set in env of RayJob or RayCluster.
```

Comment:
I saw this version of the example somewhere else that removes the need for TPU_HOSTS, would it work here?
```
import ray
import jax


@ray.remote(resources={"TPU": 4})
def my_function() -> int:
  return jax.device_count()


num_tpus = ray.available_resources()["TPU"]
num_hosts = int(num_tpus) // 4
h = [my_function.remote() for _ in range(num_hosts)]
print(ray.get(h))
```

Reply:
I just tested this with a 4x4 and it works, I'll make the change to check `ray.available_resources()["TPU"]` instead of the env vars.

Reply:
(can in a follow-up PR)

Reply:
Sounds good, I'll open one.